### ISSUE TITLE
Why are all the predicted data sets black?

### ISSUE THREAD

**Issue by zhenyangGao** [Shathe/Semantic-Segmentation-Tensorflow-Eager#11]
First of all, thank you for sharing. I use the original dataset to get the correct results, but the final predicted result image of the cityscapes dataset using RGB annotations is all black, and the output is black when I test with my own dataset. Do you know why? thank you very much!

**Comment by Shathe** [comment]
Can it be that the otput values are from the value 0 to the value n_classes? That may be the reason for seeing it black, that the values are very low

**Comment by zhenyangGao** [comment]
I use my own dataset, the input image is an RGB image, and the label is a binary image of 0 and 255.
 1. During training, I just want to try it, epoch=10, is the epoch I gave too small?
 2. The input image is RGB image is jpg format, the label is png format, is it caused by two different formats?
 3. There are only a few training sets and test sets, only 10.
 4. Does the channels variable in the loder function affect the binary image of the label?
I am still testing, I hope to get your help! Thank you
I still tested the image with the cityscapes RGB annotation, and the output is only grayed out. The previously tested dataset and epoch are relatively small. I will continue to try to increase the two and continue to try.

**Comment by zhenyangGao** [comment]
I solve it，thanks.

**Comment by Shathe** [comment]
Sorry, I had not time to reply so far. What was it?
